src/main/resources/application.yml

```
@@ -0,0 +1,23 @@
+  pattern:
+    console: "%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n"
+    file: "%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n"
+  file:
+    name: logs/application.log
+  logback:
+    rollingpolicy:
+      max-file-size: 10MB
+      max-history: 30
+      total-size-cap: 1GB
+  performance:
+    threshold-ms: 1000  # 성능 로깅 임계값 (1초)
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)